index.rst

```diff
@@ -149,7 +149,7 @@ Sensor Components
     TX20, components/sensor/tx20, tx20.jpg
     Ultrasonic Sensor, components/sensor/ultrasonic, ultrasonic.jpg
     Uptime Sensor, components/sensor/uptime, timer.svg
-    VL53L0x, components/sensor/vl53l0x, vl53l0x.svg
```

Comment:
should delete this svg file is not used anymore?

Reply:
I have deleted this file, but I found that the file `v1.14.0.rst` still use it.